File: Backend/src/Events.Domain/Event.cs
using System;
using System.Collections.Generic;

namespace Events.Domain
{
    public class Event
    {
        public int Id { get; set; }
        public string Local { get; set; }
        public DateTime? DateEvent { get; set; }
        public string Theme { get; set; }
        public int NumPeople { get; set; }
        public string UrlImage { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public IEnumerable<Batch> Batches { get; set; }
        public IEnumerable<SocialMedia> SocialMedias { get; set; }
        public IEnumerable<PanelistEvent> PanelistsEvents { get; set; }
    }
}

File: Frontend/Events-App/src/app/model/Event.ts
import { Batch } from "./Batch";
import { Panelist } from "./Panelist";
import { SocialMedia } from "./SocialMedia";

export interface Event {
  id: number
  local: string
  dateEvent?: Date
  theme: string
  numPeople: number
  urlImage: string
  phone: string
  email: string
  batches: Batch[]
  socialMedias: SocialMedia[]
  panelistsEvents: Panelist[]
}
